docs/reference/run-yaml-schema-reference.md

```diff
@@ -0,0 +1,43 @@
+| `run`                   | string        | Referenced flow run name. For example, you can run an evaluation flow against an existing run.                                                                                                                                                                          |
+| `column_mapping`        | object        | Inputs column mapping, use ${data.xx} to refer to data file columns, use ${run.inputs.xx} and ${run.outputs.xx} to refer to run inputs/outputs columns.                                                                                                                 |
+| `connections`           | object        | Overwrite node level connections with provided value. Example: --connections node1.connection=test_llm_connection node1.deployment_name=gpt-35-turbo                                                                                                                    |
+| `environment_variables` | object/string | Environment variables to set by specifying a property path and value. Example: `{"key1"="${my_connection.api_key}"}`. The value reference to connection keys will be resolved to the actual value, and all environment variables specified will be set into os.environ. |
+| `properties`            | object        | Dictionary of properties of the run.                                                                                                                                                                                                                                    |
+| `tags`                  | object        | Dictionary of tags of the run.                                                                                                                                                                                                                                          |
+| `resources`             | object        | Dictionary of resources used for automatic runtime. Only supported for cloud run. See [Resources Schema](#resources-schema) for the set of configurable properties.                                                                                                     |
+| `variant`               | string        | The variant for the run.                                                                                                                                                                                                                                                |
+| `status`                | string        | The status of the run (read-only).                                                                                                                                                                                                                                      |
```

Comment:
what does "read-only" here mean? kind of confused

Reply:
updated